### ISSUE TITLE
RuntimeError: Could not find a task with the ID 514

### ISSUE THREAD

**Issue by shreyagudapati9** [DCAN-Labs/BIBSnet#108]
### Problem Description

Hello, I am currently facing an issue while attempting to run BIBSNet using the following command inside a Singularity container:

```bash
singularity run --nv --cleanenv --no-home \
-B /path/to/BIDS:/input \
-B /path/to/output:/output \
/path/to/bibsnet.sif \
/input /output participant -v
```

### Error Encountered

The command fails with the following error message :
<img width="1675" alt="bibsnet error" src="https://github.com/DCAN-Labs/BIBSnet/assets/138166593/e15468a0-dc6a-40b0-a057-b71da66ee07a">

Could someone please provide guidance or suggestions on how to resolve this issue?

Thank you for your help!

**Comment by dnkennedy** [comment]
I also received this error...

**Comment by LuciMoore** [comment]
@shreyagudapati9 apologies for the delay - I'm not sure about this, have you tried rebuilding the sif image? perhaps try `singularity pull bibsnet.sif docker://dcanumn/bibsnet:release-3.2.0`

@tjhendrickson thoughts?

**Comment by LuciMoore** [comment]
@shreyagudapati9 our [latest release](https://github.com/DCAN-Labs/BIBSnet/releases/tag/3.4.1) has some massive improvements including updated models if you would like to try again. Please re-open this issue if you continue to run into this error!

**Comment by scott-huberty** [comment]
Re-opened at #133

**Comment by scott-huberty** [comment]
@shreyagudapati9  @dnkennedy see #134 .

For me the issue was that my user account did not have the necessary file permissions for folders/files inside the container that are needed by BIBSnet. Maybe you can try again after #134 lands and an updated image is pushed to DockerHub. 

If you _still_ get this error after that happens then please share, I would be interested to know because this error message `Could not find a task with ID` is really hard to interpret (i.e. I think many different issues could lead to this same error message), and I think we could do abetter job to try to catch these issues and raise more informative errors messages. But to do that I'd need to know what is causing the issue for you.

**Comment by dnkennedy** [comment]
Hi, I just reran my example with bibsnet:release-3.4.2.  I no longer get the "Could not find a task with ID" error, and my process proceeds farther.

I do get a different error, however, but I guess I'll raise that as a separate issue.

**Comment by scott-huberty** [comment]
@dnkennedy making Progress! 🚀

**Comment by LuciMoore** [comment]
@dnkennedy yes please post an issue!

**Comment by dnkennedy** [comment]
I posted issue https://github.com/DCAN-Labs/BIBSnet/issues/137 as my new problem.
